service-front/app/test/CommonTest/Service/Url/UrlValidityCheckServiceTest.php

```diff
@@ -11,30 +11,22 @@
 use Mezzio\Router\RouteResult;
 use Mezzio\Router\RouterInterface;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Prophecy\Prophecy\ObjectProphecy;
 use Psr\Http\Message\ServerRequestFactoryInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 class UrlValidityCheckServiceTest extends TestCase
 {
-    /**
-     * @var ServerRequestInterface
-     */
-    private $serverRequestFactoryProphecy;
-
-    /**
-     * @var RouterInterface
-     */
-    protected $routerProphecy;
-
-    /**
-     * @var ServerRequestFactoryInterface
-     */
-    protected $serverRequestInterfaceProphecy;
-
-    /**
-     * @var UrlHelper
-     */
-    private $urlHelperProphecy;
+    use ProphecyTrait;
+
+    private ServerRequestInterface|ObjectProphecy $serverRequestFactoryProphecy;
+
+    protected RouterInterface|ObjectProphecy $routerProphecy;
+
+    protected ServerRequestFactoryInterface|ObjectProphecy $serverRequestInterfaceProphecy;
+
+    private UrlHelper|ObjectProphecy $urlHelperProphecy;
 
     private $locale;
```

Comment:
```suggestion
    private ServerRequestInterface|ObjectProphecy $serverRequestFactoryProphecy;
    protected RouterInterface|ObjectProphecy $routerProphecy;
    protected ServerRequestFactoryInterface|ObjectProphecy $serverRequestInterfaceProphecy;
    private UrlHelper|ObjectProphecy $urlHelperProphecy;
    private $locale;
```

It's ok to cuddle the properties